### ISSUE TITLE
Failures running tests - test_build_latex

### ISSUE THREAD

**Issue by avylove** [sphinx-doc/sphinx#2583]
I'm trying to create an updated RPM (1.4.1 pulled from PyPi) for Fedora as the current one is based on 1.3.1. I had to add a lot of dependencies, mostly .sty files that weren't required previously. I managed to get down to one error and two failures.

The error is a parsing error in test_build_html.py, line 981, in test_enumerable_node
    etree = ET.parse(fp, parser)
    ParseError: mismatched tag: line 91, column 4

I haven't had  a chance to dig into that one too much yet. Any ideas you have are welcome.

Both failures are with test_build_latex in test_latex_howto and test_latex
Both xelatex (when installed) and lualatex return 1
It looks like they aren't properly recognizing unicode characters.
I've installed all the .sty files I can find mention of and it has helped but I have not yet gotten it to succeed

I'll attach the log and out files.
If you don't know specifically what the issue is can you confirm all the tests should succeed and which .sty files should be installed?

Thanks
[SphinxTests-latex.log.txt](https://github.com/sphinx-doc/sphinx/files/285327/SphinxTests-latex.log.txt)
[SphinxTests-latex.out.txt](https://github.com/sphinx-doc/sphinx/files/285331/SphinxTests-latex.out.txt)
[SphinxTests-latex_howto.log.txt](https://github.com/sphinx-doc/sphinx/files/285329/SphinxTests-latex_howto.log.txt)
[SphinxTests-latex_howto.out.txt](https://github.com/sphinx-doc/sphinx/files/285330/SphinxTests-latex_howto.out.txt)

**Comment by avylove** [comment]
After a lot of digging I found the problem was with '\usepackage[utf8]{inputenc}'

If I used latin1, removed the line, or saved the tex source with a different encoding, it would succeed. What I found in researching was inputenc isn't used for UTF8 in xelatex and lualatex because the compilers natively handle UTF8. When used with these compilers it results in existing control characters being treated as text.

I created a pull request for this.

Discusion of this topic can be found here:
http://tex.stackexchange.com/questions/44694/fontenc-vs-inputenc
